### ISSUE TITLE
500 Error POST /test/career-interest-profile/start_session/%5B%5D 

### ISSUE THREAD

**Issue by tejovathi-cons** [campsych/concerto-platform#269]
### Concerto Platform version
5.0.11

### Expected behavior
Created Test Must Run

### Actual behavior
After following the linearTest tutorial in ConcertoWiki, when I select the test to run, I get "Server problem encountered. Execution halted."

### Steps to reproduce the issue
1. Followed the Deployment Guide and did a bare-metal install using composer
2. Configure nginx to server PHP files through PHP-FPM using unix sockets
3. Logged in As Admin and followed the instructions in the LinearAssessment to create a Test Assessment.  In my version LinearTest was not available so selected Assessment.
4. In my version scores was not available hence used traitscores for eval node.
5. After completion, when I click on the Run Test Button, I get the error.

nginx errorlog is empty.  However, when I check the access log the following error is mentioned

122.164.67.231 - - [27/Jan/2020:17:14:55 +0530] "POST /test/career-interest-profile/start_session/%5B%5D HTTP/2.0" 500 495 "https://assess.tejovathi.com/test/career-interest-profile" "Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36"

Please let me know how I can resolve this issue.

**Comment by przemyslawlis** [comment]
If you're running 5.0.11, it means that you are not running stable release, as this version isn't released yet. Check [Deployment Guide](https://github.com/campsych/concerto-platform/wiki/deployment-guide) for more info.

You need to provide more info about your 500 error. Please find output of this request in your application logs. See [Reporting Issues](https://github.com/campsych/concerto-platform/wiki/Reporting-Issues) on how to access application logs.

**Comment by tejovathi-cons** [comment]
Dear przemyslawlis,

Thank you for addressing the issue.

I reinstalled 5.0.11 on a separate server, with a larger PHP max_postsize and isolated the database server to another virtual server.  Installed php-sessions module and it is working now.

I accessed the Dockerfile (for Ubuntu) and followed the steps in the Dockerfile (converting it to CentOS).  After which, I am able to use the assessment platform.

Thanks and will revert if I should have any further problems.

Tejovathi Consulting
